girder/utility/s3_assetstore_adapter.py

```diff
@@ -375,6 +375,7 @@ def finalizeUpload(self, upload, file):
                     'url': url,
                     'headers': headers
                 }
+                file['additionalFinalizeKeys'] = ('s3FinalizeRequest',)
```

Comment:
makes sense.  I was more concerned about adding a test if this exposed a bug, but this fixed the problem (not filtering result) and preserved behavior (returning the S3FinalizeRequest).